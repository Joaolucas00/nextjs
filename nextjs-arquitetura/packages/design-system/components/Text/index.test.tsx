import { Text } from "./index";
import { customRender } from "@alura/test-commons/react-testing-library";

const render = customRender()

describe("<Text/>", () => {
  test("Deve renderizar um h1", () => {
    const { container } = render(<Text tag="h1">Sniper Monkey</Text>);
    expect(container).toMatchSnapshot();
  });
});
