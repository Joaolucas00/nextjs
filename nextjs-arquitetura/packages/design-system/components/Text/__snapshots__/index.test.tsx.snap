// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text/> Deve renderizar um h1 1`] = `
.c0 {
  font-family: sans-serif;
}

<div>
  <h1
    class="c0"
  >
    Sniper Monkey
  </h1>
</div>
`;
